modules/contentpool_remote_register/src/RegistrationPullManagerInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drupal\contentpool_remote_register;
+
+/**
+ * Interface for RemoteAutopullManager.
+ */
+interface RegistrationPullManagerInterface {
```

Comment:
comment does not match interface. also should be named using "push" terminology --> e.g. PushManagerInterface ?